src/lib/jsonParseV2/badInput.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This file may be new but the content comes from a file from 2015 ...
Not sure what year to use in the copyright header ... (more files suffering from the same)

Reply:
In my opinion, not really important. Choose either 2015 or 2018 as you prefer.

Reply:
Yeah, I agree, not important.
I picked the year of implementation, not 'file change'

NTC

Reply:
2015 seems more correct imo